src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/HoverHandlerTest.cs

```diff
@@ -270,6 +270,35 @@ public async Task HandleRequestAsync_CSharpProjection_InvokesCSharpLanguageServe
         {
             // Arrange
             var called = false;
+
```

Comment:
Test name should change given we no longer return null in this case.

Reply:
Thanks, good catch. Updated.